### PR TITLE
elastiflow: drop unnecessary privileges on initContainer

### PR DESCRIPTION
**Pull request by majewsky**
This init container only changes file permissions on a mounted volume. It does not need to run in privileged mode as far as I can see.

### CODE REVIEW

File: system/elastiflow/values.yaml [hunk 1]
```diff
@@ -351,9 +351,3 @@ kibana:
   #     memory: 4Gi
   #   limits:
   #     memory: 5Gi
-
-# Deploy Prometheus alerts.
-alerts:
-  enabled: false
-  # Name of the Prometheus to which the alerts should be assigned to.
-  prometheus: infra-frontend
```
Comment (majewsky) [inline system/elastiflow/values.yaml:359]: This part was flagged by the CI:
```
system/elastiflow/values.yaml
  354:1     error    duplication of key "alerts" in mapping  (key-duplicates)
```
I merged it with the functionally identical section at the top of the file.

**Review by notque — APPROVED** [review]
LGTM, curious if @IvoGoman has an argument for the privileges, I'm not aware of them being needed

**Comment by majewsky** [comment]
My best guess is "copy-paste from the configure-sysctl init container". The sysctl init container does indeed need privileged access to edit kernel settings.

**Comment by notque** [comment]
Yeah, that makes sense, and it's just changing permissions, so it seems fine to just merge.

**Review by IvoGoman — APPROVED** [review]
I do not see a reason for having privileged more here.

**Pull request merged**